CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.6)
+project(tack)
+
+if (NOT MSVC)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -msse -msse2 -msse3 -msse4 -mfpmath=sse")
+endif()
+
+add_subdirectory(include)
```

Comment:
True, it felt a bit strange when I was adding this.  Its cleaner with your suggested changes having it in the root.